bitnami/apisix/crds/ApisixGlobalRule.yaml

```diff
@@ -0,0 +1,88 @@
+#
```

Comment:
Done

Reply:
Could you please use the tagged link instead of master?

That is how we know if the CRD is up-to-date or not, because when updating we would also update the URL.